Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,459 @@
+/*/*****/*********************/%%#%%%%%%%%&%**,**/(/,*(##%%#//////////**((/**/(((/**///******(%#(////((//##/,,,*,,,*,,,,,,,,*,.,/#*,,,,,..,
+/(((//**(//(/(/////***********(#####%%%%%&%(,..*,,*/,,/,,(#%%%%%%(((((((####(((/**,,,*,,,,,,,*(%%(//*/(((((*,,,,*,,*,,,,,,,,,*,,*(#,,..,,,,
+///////////////////***********/#%##%%%%%%%##//,**,*(*,/**(#&%%%##(((((((((#####(//*,,**,****/((%%#//**/((/*,,,,,,,,,,,,,,,,,,,,,,*#/,,,,..,
+////(/**(////(**//****************#######%#((#((((((/(#&&&&&&%##//////////(%%%%%##(******/((*,,***//*//****,,,,,,,,,,**,,,,.,,,,,,(#/,,,..,
+//////**///////*****************,,/*/%%/(##########(//#%%%(/**,,****,,**,*/###(/**////////******/#(***,,,,**,,,,,,,***.,,..,,,,,*%#,.,,,,
+/*/**/*//*/***********************####(#%%%%%%%%%%%%%%#(/(/**///*///*//(((*,,.,*((//////((/////(((**(((**,,,***,,*,,.,**,.,,,..,**,(%#*.,,,
+(///////(//(/(////****************##%(#%%%%%##%%%(////#((((/*/(((((/////((*,*//******///(#(////((((//(#**,,,***,,**,,,***..*,,,,,*,*(%#,,.,
+####((/((//////////*/***//*/****/*(##%%%%%%%%%&%(,,,,,/**##%%%//**********(((((//*,,**,,*/%%////((((//(,**,,,,**,,*,,,***,,,*,,,,,,,*%%/.,*
+""")
+    print("Okay \(enterName ?? "playa"), a \(howAdenturousAsAnInt) is adventurous enough to try the Vessel at Hudson Yards!!!......")
+    print("Ready to start? \"yes\" or \"no\"?")
+    
+    let readyGo = readLine()?.lowercased()
+    trophy.level = "Rockstar"
+    
+    
+    switch readyGo {
+    case "yes":
+        print("The Vessel is an extraordinary centerpiece of Hudson Yards, a soaring new landmark meant to be climbed. I'll let you take the elevator to the top if you can guess close to number of individual stairs in the Vessel..... Give it a try!!!")
+        print()
+        let flightsInStructure = readLine()
+        let flightsStructure = Int(flightsInStructure ?? "") ?? -1
+        if flightsStructure >= 2300 && flightsStructure <= 2700 {
+            print("WOW \(enterName ?? "playa")..... I'm impressed!!! you get to take the elevator to the top of \(trophy.landmark)!!!")
+            for _ in 1...5 {
+                print("🔼🔼🔼🔼🔼🔼🔼🔼")
+                sleep(1)
+            }
+            print("🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆")
+            sleep(1)
+            print("🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆")
+            print("🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆🎆")
+            print("\(enterName ?? "playa")...... Welcome to the top of The Vessel")
+            
+        } else {
+            print("Unfortunately for you \(enterName ?? "playa") this elaborate honeycomb-like structure rises 16 stories and consists of 154 flights of stairs, 2,500 steps, and 80 landings for visitors to climb")
+            print()
+            print("Looks like you're climbing")
+            print("")
+            print("How Many Stairs can you climb in one minute?")
+            let stairsPerMin = readLine()
+            let stairsPerMinInt = Int(stairsPerMin ?? "") ?? -1
```

Comment:
Changing the input and casting it as an Int and defaulting it's value is a good idea but your code should be able to handle incorrect inputs.